jwst/tests_nightly/general/nirspec/test_refpix_irs2.py

```diff
@@ -0,0 +1,42 @@
+import os
+import pytest
+from astropy.io import fits as pf
+from jwst.refpix.refpix_step import RefPixStep
+
+from ..helpers import add_suffix
+
+pytestmark = [
+    pytest.mark.usefixtures('_jail'),
+    pytest.mark.skipif(not pytest.config.getoption('bigdata'),
+                       reason='requires --bigdata')
+]
+
+
+def test_refpix_nirspec_irs2(_bigdata):
+    """
+    Regression test of refpix step performed on NIRSpec data using IRS2 readout mode.
+
+    """
+    output_file_base, output_file = add_suffix('refpix2_output.fits', 'refpix')
+
+    try:
+        os.remove(output_file)
+    except:
+        pass
+
+
+
+    RefPixStep.call(_bigdata+'/nirspec/test_bias_drift/jw84600007001_02101_00001_nrs1_superbias.fits',
+                    odd_even_columns=True, use_side_ref_pixels=True, side_smoothing_length=11,
+                    side_gain=1.0, odd_even_rows=True, output_file=output_file_base, name='refpix'
```

Comment:
What is `name`?

Reply:
It's used to set the step name that in turn gets used by stpipe to construct the output file name (it'll be the new suffix on the file name).